### DISCUSSION TITLE
PDF Export and Diagram.net?

### DISCUSSION THREAD

**Discussion by andreapx** [gollum/gollum#1958] [General]
Hi every body!
I've just discovered Gollum and I like it!
But...can it export pages in PDF? If yes, is it possible to export them with a personal header and footer?
Does has integration with Diagram.net?
I've tried to search, but due to the name...it hasn't been an easy task!  😆
Thanks!

**Comment by benjaminwil** [comment]
Hey! I don't think Gollum does PDF exports or Diagram.net

Though it may not be what you want, I wonder if you could use a custom stylesheet to provide print CSS styles and then use your browser's printing feature to get satisfactory PDFs. 

See [Custom Styling and Views](https://github.com/gollum/gollum/wiki/Custom-Styling-and-Views) on Gollum's wiki for more info.
